README.md

```diff
@@ -1,58 +1,60 @@
 ## OpenShift Wildcard Proxy
 
-Universal RHMAP platform proxy that can be used to map core and 
-mbaas requests outside OpenShift environment.
+Universal RHMAP platform proxy that can be used as an alternative to wildcard SSL certificates used in the OpenShift environment.
 
 ## Purpose of this repository
 
 Provide mapping and proxy between public nginx proxy and
-mbaas and core components running on openshift platform.
-By default proxy would use single hostname for every mbaas application 
-and I would map first element of the path into subdomain internally.
+applications deployed via RHMAP and running on OpenShift.
+
 
 ## Running on OpenShift
 
-To install proxy on openshift use supplied template
+To install Proxy on OpenShift use supplied template:
 
     oc new-app -f openshift/rhmap-nginx-proxy.json
 
-## Running on Docker
+## Running with Docker
 
-Proxy can be launched on any operating system that supports docker (Windows, Linux and MacOSX)
+Proxy can be launched on any operating system that supports Docker (Windows, Linux and MacOSX)
 
-To run service pull image from registry:
+To run the service pull the image from registry:
 
     docker pull feedhenry/wildcard-proxy-centos
 
-Run downloaded image 
+Run downloaded image
 
     docker run -it -e BASE_HOST=test-host.com -p 80:8080 feedhenry/wildcard-proxy-centos
 
 ## Environment variables
 
-
 >  BASE_HOST `required`
 
-Full base host url that would be used to proxy to the mbaas, core and apps.
-It represents openshift router dns without wildcard.
+Full base host url that would be used to proxy to the RHMAP applications.
+It represents the domain name of the OpenShift router DNS name without the first wildcard subdomain.
 
 > PLATFORM_URL `optional`
 
-Full URL to core platform. For example https://rhmap.corehost.net
+Full URL to RHMAP Core platform. For example https://rhmap.corehost.net
 
 > DNS_SERVER `optional`
 
-DNS server that would be used to query subdomains from BASE_HOST 
+DNS server that should be used to query subdomains from BASE_HOST
 For internal networks you would need to specify your local DNS server.
 
-> LOG_LEVEL `optional` 
+> LOG_LEVEL `optional`
 
 Nginx log level. By default it's INFO.
 
+## CONTRIBUTING
+
+To contribute to this repository, fork and create a pull request against master. Changes in this repository will need to be reflected in
+the OpenShift templates repositroy that are provided with the RHMAP product. Maintainers of this project are responsible for this update to [fh-core-openshift-templates](https://github.com/fheng/fh-core-openshift-templates)
```

Comment:
I think is ok to mention internal repo here. Make sure that you would have something similar with target repo.